app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingThemePickerViewHolder.kt

```diff
@@ -52,6 +52,10 @@ class OnboardingThemePickerViewHolder(private val view: View) : RecyclerView.Vie
             radioLightTheme.performClick()
         }
 
+        view.clickable_region_automatic.contentDescription =
+            view.context.getString(R.string.onboarding_theme_automatic_title) +
```

Comment:
Could we please use a Kotlin string template here? We don't want to simply concatenate or append. Adding a space is significant to how the content description is read.

Reply:
Makes sense! Updated